dashboard/src/actions/charts.test.tsx

```diff
@@ -8,8 +9,8 @@ import { NotFoundError } from "../shared/types";
 const mockStore = configureMockStore([thunk]);
 
 let store: any;
-const fetchOrig = window.fetch;
 let fetchMock: jest.Mock;
+let axiosGetMock: jest.Mock;
```

Comment:
Not a blocker for this PR, but you can easily mock external modules with jest directly :) https://jestjs.io/docs/en/mock-functions#mocking-modules

Reply:
I am not sure how to use it to simplify those test. I would still need to define axios.get as a mock and specify the logic, isn't it?

Reply:
It simplifies the current code because `axios` would be a mock already:

```js
// Instead of 
let axiosGetMock: jest.Mock;
axiosGetMock = jest.fn(() => {
  return { data: response };
});
axios.get = axiosGetMock;

// Just
jest.mock('axios')
axios.get.mockResolvedValue({ data: response })
```